realgeeks.com.website_and_crm.json

```diff
@@ -0,0 +1,69 @@
+{
+  "providerId": "realgeeks.com",
+  "providerName": "Real Geeks",
+  "serviceId": "realgeeks",
+  "serviceName": "Real Geeks Real Estate Websites",
+  "version": 1,
+  "logoUrl": "https://www.realgeeks.com/static/img/logo_2x.png",
+  "description": "Enables a domain to be used for a Real Geeks website and CRM",
+  "variableDescription": "Self explanatory in the variable names.",
+  "records": [
+    {
+      "groupId": "Website",
+      "type": "APEXCNAME",
+      "host": "@",
+      "pointsTo": "%PRODHOST%",
+      "ttl": 600
+    },
```

Comment:
APEXCNAME will severely limit you on DNS Providers you can work with; but this is your perogative.

Also recommend that you make variable names lower case....experience tells me this is a bit more friendly.

You also are going to want to digitally sign this template.  A synchronous template with a variable in pointsTo is susceptible to fishing attacks.  See section 4.2.3 of the spec

Reply:
The APEXCNAME is actually one of the requirements we had for using Domain Connect. Luckily GoDaddy supports it, and they're currently the only ones we're wanting to work with.

The caps lock variables felt bad to type honestly.

This is meant to be a synchronous template, so it definitely needs to be signed. Thanks for the input!

Reply:
Added the syncPubKeyDomain key to the template.